lib/menu.dart

```diff
@@ -263,7 +270,8 @@ class _MenuState extends State<Menu> {
                                                   ),
                                                   onPressed: () {
                                                     setState(() {
-                                                      currentScreen = About();
+                                                      currentScreen = About(
```

Comment:
why do you get the version as parameter and not use it in the about page initialization?

Reply:
Because it's an async operation and `initState()` is not async and I didn't want to add a `FutureBuilder` there